addons/social_share/wizards/snshare_url_share_multi.py

```diff
@@ -0,0 +1,30 @@
+from markupsafe import Markup
+
+from odoo import fields, models
+
+
+class ShareWizard(models.TransientModel):
+    _name = 'snshare.url.share.multi'
```

Comment:
Did not try this one, but not sure to understand how it works (where are "urls" generated ?)